examples/werkstatt/werkstatt_darksat.js

```
@@ -0,0 +1,58 @@
+
+      // Tone: shelving (lowpass/highpass blend)
+      this.lpL = this.lpL * 0.7 + satL * 0.3
+      this.lpR = this.lpR * 0.7 + satR * 0.3
+      this.hpL = satL - this.lpL
+      this.hpR = satR - this.lpR
+      const tonedL = this.lpL * (1 - toneMix) + this.hpL * toneMix
+      const tonedR = this.lpR * (1 - toneMix) + this.hpR * toneMix
+
+      // Dry/wet + output gain
+      io.out[0][i] = (inL * (1 - wetMix) + tonedL * wetMix) * outL
+      io.out[1][i] = (inR * (1 - wetMix) + tonedR * wetMix) * outR
```

ESLint

[error] 'outR' is not defined.

(no-undef)